conu/backend/docker/image.py

```diff
@@ -221,6 +227,118 @@ def push(self, repository=None, tag=None):
                                         self.name, error)
         return image
 
+    def using_transport(self, transport=None, path=None, logs=True):
```

Comment:
I would argue about the naming here, `change_transport` or even previous `set_transport` makes more sense here for me.

Reply:
Good point, LGTM from my side.